openapi.yml

```
@@ -0,0 +1,268 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CPUMetricResponse'
+        '207':
+          description: Multi-Status | Some of the data is not available
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CPUMetricResponse'
+      security:
+          - bearerAuth: []
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)